setup.py

```diff
@@ -18,6 +18,7 @@
         "pycountry==19.8.18",
         "phonenumberslite==8.9.15",
         "requests==2.22.0",
+        "factory-boy~=2.12.0",
```

Comment:
We're only using this in tests, so it should be added to `requirements-dev` instead